src/solc_0.8/asset/ERC1155ERC721.sol

```diff
@@ -774,6 +774,36 @@ contract ERC1155ERC721 is WithSuperOperators, IERC1155, IERC721, ERC2771Handler
         }
     }
 
+    function _burnBatch(
+        address from,
+        uint256[] memory ids,
+        uint256[] memory amounts
+    ) internal {
+        address sender = _msgSender();
```

Comment:
minor, but caching the return value of `_msgSender()` here isn't needed. I usually apply this pattern to avoid multiple calls to `_msgSender()` in the same function, but here it is only called once.

Reply:
Yeah makes sense :+1: I've resolved it